lib/src/get_position.dart

```diff
@@ -40,10 +43,14 @@ class GetPosition {
         box.size.bottomRight(box.localToGlobal(const Offset(0.0, 0.0)));
 
     final rect = Rect.fromLTRB(
-      topLeft.dx,
-      topLeft.dy,
-      bottomRight.dx,
-      bottomRight.dy,
+      topLeft.dx - padding!.left < 0 ? 0 : topLeft.dx - padding!.left,
```

Comment:
Are you sure that the `padding` won't be null here?

Reply:
I have updated the PR. please review the changes.